.github/workflows/main.yml

```diff
@@ -94,7 +94,7 @@ jobs:
     strategy:
       matrix:
         python-version: [ '3.7', '3.9', '3.10' ]
-        os: ['macos-10.15']
+        os: ['macos-11', 'macos-12']
```

Comment:
With the scarcity of macOS runners, I'd be in favor of limiting the matrix to just one macOS version.

Reply:
Yeah, and especially since the tests passed from this PR on macOS 12, I'm happy to remove it.